src/test/kotlin/org/malachite/estella/people/HrPartnerIntegration.kt

```diff
@@ -39,6 +43,33 @@ class HrPartnerIntegration : BaseIntegration() {
         withStatusAndMessage(response, "Address already in use!", HttpStatus.BAD_REQUEST)
     }
 
+    @Test
+    @Order(4)
+    fun `should return ok and list of partner offers`(@Autowired offerService: OfferService) {
```

Comment:
nie wiem co myslec o tym serwisie tutaj, ale moze można xD

Reply:
Wygląda dosyć hackersko XD, ogólnie myślałem nad tym żeby mieć dostęp z adminApiKey do wszystkich endpointów i wtedy moglibyśmy w testach po prostu wysyłać zapytania na endpoint z headerem że to adminApiKey, ale to jest do przegadania czy chcemy w ogóle taką funkcjonalność